paths_cli/compiling/core.py

```diff
@@ -0,0 +1,300 @@
+import json
+import yaml
+
+from collections import namedtuple
+
+import logging
+
+from .errors import InputError
+from paths_cli.utils import import_thing
+from paths_cli.plugin_management import OPSPlugin
+
+def listify(obj):
+    listified = False
+    if not isinstance(obj, list):
+        obj = [obj]
+        listified = True
+    return obj, listified
```

Comment:
Is this intended to wrap tuples? or only things that are not a `collections.abc.Sequence`?

Reply:
Since the input is coming from a JSON-compatible parser, the only possibility is a `list`.